configure

```diff
@@ -30,6 +30,7 @@ valid_os = ('win', 'mac', 'solaris', 'freebsd', 'openbsd', 'linux',
 valid_arch = ('arm', 'arm64', 'ia32', 'mips', 'mipsel', 'ppc', 'ppc64', 'x32',
               'x64', 'x86')
 valid_arm_float_abi = ('soft', 'softfp', 'hard')
+valid_arm_fpu = ('vfp', 'vfpv2', 'vfpv3', 'neon')
```

Comment:
V8 also knows about vfpv3-d16.  I don't know how common it is but might be nice to add.